.github/workflows/comprehensive_tests.yml

```
@@ -0,0 +1,329 @@
+      - name: Run ESLint
+        working-directory: telegram-mini-app
+        run: npm run lint || true
+      
+      - name: TypeScript type check
+        working-directory: telegram-mini-app
+        run: npx tsc --noEmit
+      
+      - name: Run unit tests with coverage
+        working-directory: telegram-mini-app
+        run: npm run test:unit -- --coverage
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)